pcsx2/DEV9/AdapterUtils.cpp

```diff
@@ -244,30 +246,93 @@ std::optional<MAC_Address> AdapterUtils::GetAdapterMAC(Adapter* adapter)
 
 	return std::nullopt;
 }
-#elif defined(__POSIX__)
-#ifdef __linux__
+#else
 std::optional<MAC_Address> AdapterUtils::GetAdapterMAC(Adapter* adapter)
 {
-	struct ifreq ifr;
-	strcpy(ifr.ifr_name, adapter->ifa_name);
-
-	int fd = socket(AF_INET, SOCK_DGRAM, 0);
-	int ret = ioctl(fd, SIOCGIFHWADDR, &ifr);
-	close(fd);
+	int sd;
```

Comment:
Please don't put all the definitions at the top of the function, this is C++, not C89.
Declare the variable at the same place it is assigned. Helps readability.

Reply:
For sure!